src/app/api/workers/scrap-images/scrap-images.controller.ts

```diff
@@ -0,0 +1,15 @@
+import { NextRequest, NextResponse } from 'next/server'
+import { ScrapImageService } from '@/app/api/workers/scrap-images/scrap-images.service'
+import { cronSecret } from '@/config'
+
+export const RemoveScrapImages = async (request: NextRequest) => {
```

Comment:
Functions should be camelCased. Also please wrap this with withErrorHandler for proper error logging